scripts/run_integration_tests_aarch64.sh

```diff
@@ -252,11 +252,21 @@ update_workloads() {
         echo "bar" > "$SHARED_DIR/file3" || exit 1
     fi
 
-    # checkout and build SPDK NVMe
-    build_spdk_nvme
+    # Check and build SPDK NVMe
+    NVMF_TGT="$WORKLOADS_DIR/spdk-nvme/nvmf_tgt"
+    RPC_PY="$WORKLOADS_DIR/spdk-nvme/rpc.py"
+    RPC="$WORKLOADS_DIR/spdk-nvme/rpc"
+    if [[ ! -f "$NVMF_TGT" || ! -f "$RPC_PY" || ! -d "$RPC" ]]; then
```

Comment:
This logic doesn't work correctly if the source code version is bumped.

Reply:
indeed, I will fix that

Reply:
done